cmd/autotestgridgenerator/main.go

```diff
@@ -98,9 +99,26 @@ func main() {
 		logrus.Info("no new commits, exiting ...")
 		os.Exit(0)
 	}
```

Comment:
Whats the reason to copy all of this code instead of just using the `UpsertPR` functionality from the prcreation package?

Reply:
Thats mostly because most of the functionality was already checked in before I was aware of this module and adding the fork creation logic in here seemed like the faster approach, but you are right I should reuse the UpsertPR function.